### DISCUSSION TITLE
is it possible to have Daemon for wezterm?

### DISCUSSION THREAD

**Discussion by Monirzadeh** [wezterm/wezterm#2153] [Q&A]
Hi
I have small problem with wezterm
if I have a wezterm terminal, any new terminal open up so much faster.
```
wezterm  0.03s user 0.03s system 92% cpu 0.064 total

```
But first terminal always open slower 
```
 0.25s user 0.10s system 46% cpu 0.754 total
```
I am not sure why exactly it is happening, but is it possible to run a terminal in background without gui?. Something like a daemon?

Update:
i set this to config but not work
https://github.com/wez/wezterm/blob/7deb215303ce1a8e64d48f65ef3e0a1d24fc2fbc/docs/config/lua/config/daemon_options.md

**Comment by wez** [comment]
The default mode is for subsequently launched wezterms to open new windows via the first opened gui process.

You can run a multiplexer server if you wish; take a look at: https://wezfurlong.org/wezterm/multiplexing.html

**Reply by Monirzadeh** [reply]
i get a little confuse about that
i should add 
```
unix_domains = {
    {
      name = "unix",
    }
},
default_gui_startup_args = {"connect", "unix"},
```
To the config, right?
Then wezterm create a server name `unix` and when I run wezterm next time it automatically connects to that server.
But I get this error 
```
ERROR  wezterm_gui::termwindow > PaneOutput: wanted mux_window_id=0 from mux, but was not found, cancel mux subscription
```
